app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -59,12 +59,16 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
     .shape({
       label: yup.string().oneOf(CONTACTS.map(contact => contact.label)),
       value: yup.string().oneOf(CONTACTS.map(contact => contact.value)),
-    }),
+    })
+    .required("Contact is required"),
   tags: yup
-    .object()
+    .array()
     .nullable()
```

Comment:
Tags should not be nullable. Since tags hold an array of items, the empty state should be an empty array `[]`. Modify your initial values accordingly.

Reply:
Sure Yedhin, Thanks for the information.